source/fundamentals/geo.txt

```diff
@@ -0,0 +1,332 @@
+.. _golang-geo:
+
+=========================
+Work with Geospatial Data
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to work with **geospatial data**; data formats,
+indexes, and queries. Geospatial
+data represents a geographic location on the surface of the Earth, or data
+on a Euclidean plane.
+
+Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+- Points on a graph
+
+Store Geospatial Data
+---------------------
+
+All geospatial data in MongoDB is stored in one of the following formats:
+
+- GeoJSON, a format that represents geospatial data on an earth-like
+  sphere.
+
+- Legacy Coordinate Pair, a format that represents geospatial data
+  on a Euclidean plane.
+
+GeoJSON
+~~~~~~~
+
+GeoJSON is the preferred format to store geospatial data in MongoDB.
+To specifying a location you must declare one or more **positions**
+and choose the appropriate **type**.
+
+Positions
+`````````
+
+A position represents a single place on Earth and exists in code as an array
+containing the following values:
+
+- Longitude in the first position (required)
+- Latitude in the second position (required)
+- Elevation in the third position (optional)
+
+The following is the **position** of the MongoDB Headquarters in
+New York City, NY.
+
+.. code-block:: go
+   :copyable: False
+
+   []float64{-73.986805, 40.7620853}
+
+.. important:: Longitude then Latitude
+
+  GeoJSON orders coordinates as **longitude** first and **latitude** second.
+  This may be surprising as geographic coordinate system conventions generally list
+  latitude first and longitude second. Make sure to check what format any other
+  tools you are working with use. Popular tools such as OpenStreetMap and Google
+  Maps list coordinates as latitude first and longitude second.
+
+
+Types
+`````
+
+Your GeoJSON object's type determines its geometric shape. Geometric shapes are
+made up of positions.
+
+Here are some common GeoJSON types and how you can specify them with positions:
+
+- ``Point``: a single position. This could represent the location of the
+  MongoDB Headquarters.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "HQ"},
+         {"location", bson.D{
+             {"type", "Point"},
+             {"coordinates", []float64{-73.986805, 40.7620853}},
+         }},
+     }
+
+- ``LineString``: an array of two or more positions that forms a series of line
+  segments. This could represent
+  `the route of the Great Wall of China <https://commons.wikimedia.org/wiki/File:GreatWallChina4.png>`__.
+
```

Comment:
Suggestion: Make link between intro sentence and GeoJSON object more explicit

Issue (Mike): "could represent" -> "represents" - [avoid subjunctive mood](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/#avoid-the-subjunctive-mood)

Applies to multiple examples

```suggestion
  segments. The following GeoJSON object represents
  `the route of the Great Wall of China <https://commons.wikimedia.org/wiki/File:GreatWallChina4.png>`__:

```

Reply:
The style guide contains guidelines, not hard and fast rules. The main point is that a linestring is an array of two or more positions that forms a series of line segments. The subjunctive is *what* it could represent, which is anything, such as the great wall of china, broadway, the apian way, etc...

Reply:
I think this comment is more about the fact that the example below does in fact represent a portion of the route of the great wall of china, where the text is a hypothetical (not making the link between the text and the presented snippet explicitly clear).

Maybe:

This could represent a route or path. The following GeoJSON object represents the... : 

This isn't super important, so I won't have this block a merge. Mike and I both felt the current phrasing could be more effective if updated.